### PR TITLE
Ribotaper

### PR DESCRIPTION
**Pull request by dyusuf**

### CODE REVIEW

**Comment by dyusuf** [comment]
Hi Bjoern, the travis test would pass because the following conditions would not be met on the testing system:

- Please add the following line to **datatypes_conf.xml** to enable the data type, **compressed_archive**,

    <datatype extension="compressed_archive" type="galaxy.datatypes.binary:CompressedArchive" subclass="True" display_in_upload="True"/>

- At least 2 cores should be used for running ``ribotaper part 3: ribosome profiling``,  therefore please configure **job_conf.xml** accordingly.

Could you just merge the changes?

File: tools/rna_tools/ribotaper/ribotaper_part1_create_annotation_files.xml [hunk 1]
```diff
@@ -87,6 +87,24 @@ exon sequences and transcript structures.
 is used as an input for ``ribotaper part 2``.
 *Annotation_path*
 is used as an input for ``ribotaper part 3``.
+
+Important notes
```
Comment (bgruening) [inline tools/rna_tools/ribotaper/ribotaper_part1_create_annotation_files.xml]: I don't think we need this and it should probably not in the tool XML file. This is information for an admin and should be in a repository README.rst file.

File: tools/rna_tools/ribotaper/ribotaper_part2_create_metaplots.xml [hunk 2]
```diff
@@ -86,6 +86,23 @@ From the output, user shall determine the appropriate
 read lengths and cutoffs which are required
 for running ``ribotaper part 3``, ribosome profiling.
 
+Important notes
+----------------
+
+- To install the galaxy tool, please enable conda installation in **galaxy.ini**
+
+- Please add the following line to **datatypes_conf.xml** to enable the data type, **compressed_archive**,
```
Comment (bgruening) [inline tools/rna_tools/ribotaper/ribotaper_part2_create_metaplots.xml]: Some here.

File: tools/rna_tools/ribotaper/ribotaper_part1_create_annotation_files.xml [hunk 3]
```diff
@@ -87,6 +87,24 @@ exon sequences and transcript structures.
...
+----------------
+
+- To install the galaxy tool, please enable conda installation in **galaxy.ini**
+
+- Please add the following line to **datatypes_conf.xml** to enable the data type, **compressed_archive**,
+
+    <datatype extension="compressed_archive" type="galaxy.datatypes.binary:CompressedArchive" subclass="True" display_in_upload="True"/>
```
Comment (bgruening) [inline tools/rna_tools/ribotaper/ribotaper_part1_create_annotation_files.xml]: Can you simply use `tar` or `zip`?

**Comment by bgruening** [comment]
ping @dyusuf 
Would be nice to have this in the Falvour by SAB.

**Comment by dyusuf** [comment]
then merge it man :)

**Comment by dyusuf** [comment]
i just want get this tool off my neck. it is pain to wrap it

File: tools/rna_tools/ribotaper/ribotaper_part1_create_annotation_files.xml (hunk 3 above)
Comment (dyusuf) [inline tools/rna_tools/ribotaper/ribotaper_part1_create_annotation_files.xml]: no. i don't want to play with it any more

Comment (dyusuf) [inline tools/rna_tools/ribotaper/ribotaper_part1_create_annotation_files.xml]: I don't remember exactly why using compressed_archive. I think we talked about it. compressed_archive could the inspration of u

Comment (dyusuf) [inline tools/rna_tools/ribotaper/ribotaper_part1_create_annotation_files.xml]: @bgruening I tested tar vs zip, tar is a better choice. 

do u want me to make a new pull request or you could change zip to tar?

In addition, I don't find this version in the main toolsheld where has an older version.

**Comment by bgruening** [comment]
please open a PR, I will push to the MTS

**Pull request merged**